docs/traceflow-guide.md

```diff
@@ -110,7 +119,54 @@ spec:
 The CRD above starts a new trace from source Pod named `tcp-sts-0` to destination Pod named `tcp-sts-2` using ICMPv6
 protocol.
 
-### Using antctl and spec config
+### Live-traffic Traceflow
+
+Starting from Antrea version 1.0.0, you can trace a packet of the real traffic
+from or to a Pod, instead of the injected packet. To start such a Traceflow, add
+`liveTraffic: true` to the Traceflow `spec`. Then the first packet of the first
+connection that matches the Traceflow spec will be traced, and the headers of
+the packet will captured and reported to the `status` field of the Traceflow
```

Comment:
reported in the `status` field of the Traceflow CRD, in addition to the observations

nit: `status` is capitalized earlier in the doc

Reply:
Since it is `status` in the yaml, I changed the capitalized one to `status` too.